unittest/force-styles/tests/mol-pair-lj96_cut.yaml

```diff
@@ -0,0 +1,90 @@
+---
+lammps_version: 21 Jul 2020
+date_generated: Thu Jul 30 17:50:31 202
+epsilon: 5e-14
+prerequisites: ! |
+  atom full
+  pair lj96/cut
+pre_commands: ! ""
+post_commands: ! |
+  pair_modify mix arithmetic
+input_file: in.fourmol
+pair_style: lj96/cut 8.0
+pair_coeff: ! |
+  1 1 0.03375   2.5
+  2 2 0.0084375 1
+  2 4 0.0084375 0.5
+  3 3 0.03375   3.2
+  4 4 0.0253125 3.1
+  5 5 0.0253125 3.1
+extract: ! ""
```

Comment:
Lj96 doesn't have any "extract" function. Is that a concern?

Reply:
No. Those are usually added as needed. You may add those (and the tests for their existence), but then you should also update the tables in the fix adapt and fix adapt/fep tables listing compatible pair styles. But that would best be placed into a separate pull request and done for a larger selection of suitable pair styles (and then you could also review if entries are missing and update them as needed).